core/src/test/scala/org/apache/spark/deploy/master/PersistenceEngineBenchmark.scala

```diff
@@ -60,15 +61,23 @@ object PersistenceEngineBenchmark extends BenchmarkBase {
     runBenchmark("PersistenceEngineBenchmark") {
       val benchmark = new Benchmark(s"$numWorkers Workers", numWorkers, output = output)
 
-      benchmark.addCase("ZooKeeperPersistenceEngine", numIters) { _ =>
+      benchmark.addCase("ZooKeeperPersistenceEngine with JavaSerializer", numIters) { _ =>
```

Comment:
How about
```
Seq(serializerJava, serializerKryo).foreach { serializer =>
  benchmark.addCase("ZooKeeperPersistenceEngine with ${serializer.getClass.getSimpleName}", numIters) { _ =>
    val engine = new ZooKeeperPersistenceEngine(conf, serializer)
    ...
}
```

Reply:
Sorry, but this is also intentional for the ordering like the following.

```
OpenJDK 64-Bit Server VM 17.0.9+9-LTS on Linux 5.15.0-1051-azure
AMD EPYC 7763 64-Core Processor
1000 Workers:                                    Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
-------------------------------------------------------------------------------------------------------------------------------
ZooKeeperPersistenceEngine with JavaSerializer            1202           1298         138          0.0     1201614.2       1.0X
ZooKeeperPersistenceEngine with KryoSerializer             951           1004          48          0.0      950559.0       1.3X
FileSystemPersistenceEngine with JavaSerializer            212            217           6          0.0      211623.2       5.7X
FileSystemPersistenceEngine with KryoSerializer             79             81           2          0.0       79132.5      15.2X
BlackHolePersistenceEngine                                   0              0           0         30.9          32.4   37109.8X
```

Reply:
Oh, for this one, it seems that I misunderstood your previous comment. Let me address it at the next PR (or a follow-up), @beliefer . Sorry for my mistake.